packit/utils/bodhi.py

```diff
@@ -27,4 +30,9 @@ def get_bodhi_client(fas_username: str = None, fas_password: str = None) -> Bodh
         f'Initialisation of the Bodhi client: FAS user {fas_username or "not configured"}; '
         f'password {"provided" if fas_password else "not configured"}'
     )
-    return BodhiClient(username=fas_username, password=fas_password)
+    return BodhiClient(
+        username=fas_username,
+        password=fas_password,
+        cache_session=bool(fas_username and fas_password),
```

Comment:
Based on the comment I'd expect to see
```suggestion
        cache_session=not bool(fas_username and fas_password),
```

Reply:
Ah, well spotted!